docs/source/user/application-structure.rst

```diff
@@ -3,3 +3,143 @@ Application Structure
 
 .. contents:: |toctitle|
     :local:
+
+.. _accuracy:
+
+Accuracy
```

Comment:
Maybe call this Determining Accuracy?

Also, I think I'd move this section after the binary search section.